public/utils/constants.ts

```diff
@@ -51,7 +55,11 @@ export const BREADCRUMBS = Object.freeze({
     text: 'Create recipient group',
     href: `#${ROUTES.CREATE_RECIPIENT_GROUP}`,
   },
-  EDIT_RECIPIENT_GROUP: { text: 'Edit recipient group' },
+  EDIT_RECIPIENT_GROUP: { text: 'Edit recipient group', href: `#${ROUTES.EDIT_RECIPIENT_GROUP}`},
+  EDIT_RECIPIENT_GROUP_DETAILS: (name: string) => ({
+    text: `${name}`,
```

Comment:
Can be simplified to name, don't need the string literal

Reply:
Same in other places

Reply:
addressed